pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -643,6 +649,12 @@ func TestZeroRequest(t *testing.T) {
 							strconv.FormatInt(priorityutil.DefaultMemoryRequest*3, 10)),
 					},
 				},
+				ResourcesAllocated: v1.ResourceList{
+					v1.ResourceCPU: resource.MustParse(
+						strconv.FormatInt(priorityutil.DefaultMilliCPURequest*3, 10) + "m"),
```

Comment:
why 3 and 10?

Reply:
this is set from the above resources line. we need to set it because the podresourceallocation plugin defaults the ResourcesAllocated to Resources.Requests if not set per the new KEP design.